crates/ide/src/syntax_highlighting/tests.rs

```diff
@@ -1126,5 +1126,5 @@ fn benchmark_syntax_highlighting_parser() {
             .filter(|it| it.highlight.tag == HlTag::Symbol(SymbolKind::Function))
             .count()
     };
-    assert_eq!(hash, 1608);
```

Comment:
This seems odd, we effectively resolve less things as functions in this test now? Can you make sure that analysis-stats didn't regress?

Reply:
Oh actually, this might be because this test runs without the sysroot which this PR might cause to regress some resolutions depending on things in there (which is fine, but verifying analysis-stats would be good if you haven't yet)